server/cmd/run.go

```diff
@@ -55,12 +56,16 @@ func run(ctx context.Context, c *config.Config) error {
 		return err
 	}
 
+	log.Info("Setting up cleaner...")
+	cleaner := cleaner.NewCleaner(st, c.Cleaner.RetentionPeriod, c.Cleaner.PollInterval, logger)
```

Comment:
Can you add a TODO comment on having a separate deployment for this? When we have more than one replica, we don't want to run `cleaner` from every pod.